<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLDRichTextEditTableValueTest</class>
 <widget class="QWidget" name="GLDRichTextEditTableValueTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GLDRichTextEditTableValueTest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>表:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtTable"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>行:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtRow"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>列:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtColumn"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>值:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtValue"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>初始值:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtInit"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnInt">
       <property name="text">
        <string>设置表初始值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGet">
       <property name="text">
        <string>获取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSet">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
